modules/proc_creating-aggregated-cluster-role.adoc

```diff
@@ -78,6 +79,7 @@ that represent the permissions that are granted to the role. For example, apply
 read and write permissions to the admin and edit roles and only read permission
 to the view role. 
 <8> Specify this label to grant permissions to the view default role. 
+<9> Specify this label to grant permissions to the cluster-reader default role.
```

Comment:
should cluster-reader have mark-up, such as *cluster-reader*?

Reply:
For the 3.x stream, yes.